src/openfermion/hamiltonians/_jellium.py

```diff
@@ -268,12 +147,14 @@ def dual_basis_jellium_model(grid, spinless=False,
         kinetic (bool): Whether to include kinetic terms.
         potential (bool): Whether to include potential terms.
         include_constant (bool): Whether to include the Madelung constant.
+            Note: constant is unsupported for non-uniform, non-cubic cells with
```

Comment:
See my comment below: I think that the way you've written this note will confuse the sphinx documentation.